crates/yakui-vulkan/src/lib.rs

```diff
@@ -388,6 +388,7 @@ impl YakuiVulkan {
     ///
     /// ## Safety
     /// - `vulkan_context` must be the same as the one used to create this [`YakuiVulkan`] instance
+    /// - `cmd` must be in rendering state, with viewport and scissor dynamic states set.
```

Comment:
I was going to quibble on this one with "well we should update the docs to tell the developer they need to.." but unfortunately you were one step ahead of me. 😛